templates/win_userdata.ps1

```diff
@@ -198,6 +199,20 @@ function Open-WinRM {
         vtype='REG_DWORD'
     Write-Tfi "Command [salt-call --local -c C:\Watchmaker\salt\conf ash_lgpo.set_reg_value key='AllowUnencryptedTraffic'...]" $?
   }
+  elseif (Test-Path -path "C:\Program Files\Salt Project\salt\salt-call.bat") {
```

Comment:
instead of duplicating all the logic, can we set a var for salt-call? e.g.

```
if (Test-Path -path "C:\salt\salt-call.bat") {
    $SaltCall = "C:\salt\salt-call.bat"
}
elseif (Test-Path -path "C:\Program Files\Salt Project\salt\salt-call.bat") {
    $SaltCall = "C:\Program Files\Salt Project\salt\salt-call.bat"
}

if ($SaltCall -ne $null) {
    # ... current logic
    & $SaltCall ...
}
```